packages/salesforcedx-vscode-core/src/commands/index.ts

```diff
@@ -135,11 +135,7 @@ export {
   turnOffLogging,
   ForceStopApexDebugLoggingExecutor
 } from './forceStopApexDebugLogging';
-export {
-  forceAuthLogoutAll,
-  ForceAuthLogoutAll,
-  forceAuthLogoutDefault
-} from './auth/forceAuthLogout';
+export { orgLogoutAll, orgLogoutDefault, OrgLogoutAll } from './auth/orgLogout';
```

Comment:
I find this line to be confusing because the class OrgLogoutAll and the function orgLogoutAll share the same name other than the capitalization.  I know Typescript is case sensitive and that originally there were ForceAuthLogoutAll and forceAuthLogoutAll but I do wish the distinction could be clearer.

I'm thinking something like changing orgLogoutAll to orgLogoutAllFunction just to tell readers that the code is referring to the function and not the class.  Or changing OrgLogoutAll to OrgLogoutAllClass so that you don't need to change orgLogoutDefault.  I'm just hoping to see variable names that differ by more than just capitalization.